test/e2e/resources/pipeline-maven.yaml

```diff
@@ -48,6 +47,8 @@ spec:
           value: "false"
         - name: VERBOSE
           value: "$(params.VERBOSE)"
+        - name: SUBDIRECTORY
+          value: $(context.pipelineRun.name)
```

Comment:
what if subdirectory is npt given will the woking behavior is same ?

Reply:
yes it is, checked that as well, i felt that our tests should have all params so ya, could include 2 tests if u feel thats needed, but yes i did check them out, it works as previously when Subdirectory is not populated.